File: beginner/pagina08/2540/2540.ts
import * as fs from "fs"
const input = fs.readFileSync("/dev/stdin", "utf8").trim().split("\n")
const casos = input.length / 2;

for(let i = 0; i < casos; i++){

    const jogadores = Number(input.shift())
    const doisTercos = jogadores * (2 / 3)
    const votos = input.shift().split(" ").map(Number)
    let votosImpeachment = 0

    for(let j = 0; j < jogadores; j++){
        votosImpeachment += votos[j] === 1 ? 1 : 0
    }

    console.log(votosImpeachment >= doisTercos ? "impeachment" : "acusacao arquivada")
}

File: beginner/pagina08/2540/2540.java
import java.util.Scanner;

public class Main {
    public static void main (String[] args) {

        Scanner in = new Scanner(System.in);

        while (in.hasNextInt()) {

            int jogadores = in.nextInt();
            in.nextLine();

            String[] votos = in.nextLine().split(" ");
            int votosImpeat = 0;

            for(int i = 0; i < jogadores; i++){
                if(Integer.parseInt(votos[i] == 1)){
                    votosImpeat++;
                }
            }

            double doistercos = jogadores * (2.0 / 3.0);

            System.out.println(votosImpeat >= doistercos ? "impeachment" : "acusacao arquivada");

        }

        in.close();
    }
}